plugins/woocommerce/includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -152,7 +152,11 @@ public function track_product_updated_client_side( $post ) {
 
 					if ( ! isBlockEditor ) {
 						tagsText          = $( '[name=\"tax_input[product_tag]\"]' ).val();
-						description_value  = $( '#content' ).is( ':visible' ) ? $( '#content' ).val() : tinymce.get( 'content' ).getContent();
+						if ( $( '#content' ).is( ':visible' ) ) {
+							description_value = $( '#content' ).val();	
+						} else if ( typeof tinymce === 'object' ) {
```

Comment:
Sorry for the quick double check here, but I think we also want to add this condition `tinymce.get( 'content' )`
The way the error read in the original issue, made it sound like that `tinymce.get( 'content' )` was actually returning  `null` and `tinymce` was available.
But I think the `tinymce` check is still helpful.

Reply:
Great point @louwie17 thanks for your advice. Could you take a look now please?